bin/rails

```diff
@@ -1,5 +1,29 @@
 #!/usr/bin/env ruby
-load File.expand_path("spring", __dir__)
-APP_PATH = File.expand_path('../config/application', __dir__)
-require_relative "../config/boot"
-require "rails/commands"
+# frozen_string_literal: true
```

Comment:
why those files under `bin` directory are added/changed ?

Reply:
> 

It does it by itself automatically